source/transactions/tests/write-concern.yml

```diff
@@ -242,3 +230,327 @@ tests:
         result:
           # Client-side error.
           errorContains: transactions do not support unacknowledged write concern
+
+  - description: unacknowledged write concern coll insertOne
+
+    operations:
+      - *startTransaction
+      - name: insertOne
+        object: collection
```

Comment:
This is redundant. `<<` below already injects `object: collection`.

Reply:
Done.